gateway/rdb/src/main/kotlin/org/doorip/gateway/rdb/trip/TripJpaEntity.kt

```
@@ -0,0 +1,35 @@
+@Table(name = "trip")
+@Entity
+internal class TripJpaEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "trip_id", columnDefinition = "bigint", nullable = false)
+    var id: Long = 0
+        protected set
+
+    @Column(name = "code", columnDefinition = "varchar(255)", nullable = false)
+    lateinit var code: String
+        protected set
```

detekt (1.23.7)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)